components/org.wso2.carbon.identity.scim2.common/src/test/java/org/wso2/carbon/identity/scim2/common/utils/SCIMConfigProcessorTest.java

```diff
@@ -0,0 +1,91 @@
+package org.wso2.carbon.identity.scim2.common.utils;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.wso2.charon3.core.exceptions.CharonException;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SCIMConfigProcessorTest {
+    private SCIMConfigProcessor scimConfigProcessor;
+    private AuthenticationSchema authenticationSchema;
+
+    @DataProvider(name = "facebook-authenticator-all") public static Object[][] propertyProvider() {
+        return new Object[][] { { "Name", "aaaaa" }, { "Address", null } };
+    }
+
+    @DataProvider() public static Object[][] filePathProvider() {
+        return new Object[][] { { "/home/shan" }, { "/shan" },
+                { "/home/shan/testing_project/identity-inbound-provisioning-scim2/components/org.wso2.carbon.identity.scim2.common/src/test/resources/charon-config-tst.xml" } };
```

Comment:
Use the relative path to the resources directory.

Reply:
done